Application Actions/Preview/1.0/default.json

```diff
@@ -0,0 +1,9 @@
+{
+  "RotateLeft"  : "Scroll Down",
+  "RotateRight" : "Scroll Up",
+  "ButtonPress" : "Toggle Full Screen",
+  "SwipeUp"     : "Home",
+  "SwipeDown"   : "End",
+  "SwipeLeft"   : "Page Up",
+  "SwipeRight"  : "Page Down"
+}
```

Comment:
Can we make names a little more concise and say

Home -> Scroll to Beginning
End -> Scroll to End
Scroll 1 Page Up
Scroll 1 Page Down

What do you think?